src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubBuildStatusNotification.java

```diff
@@ -55,13 +55,10 @@
 import org.kohsuke.github.GHRepository;
 import org.kohsuke.github.GitHub;
 
-
 /**
  * Manages GitHub Statuses.
  *
- * Job (associated to a PR) scheduled: PENDING
- * Build doing a checkout: PENDING
- * Build done: SUCCESS, FAILURE or ERROR
+ * Job (associated to a PR) scheduled: PENDING Build doing a checkout: PENDING Build done: SUCCESS, FAILURE or ERROR
```

Comment:
This is mangled.

Reply:
should really have been in a definition list or the like (eclipse here is at least happy about not reformatting that)

Reply:
Yeah, this highlights false assumption on the part of the person writing the javadoc.